content/en/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md

```diff
@@ -62,16 +62,28 @@ upgrade the control plane nodes before upgrading your Windows nodes.
     kubeadm upgrade node
     ```
 
-### Upgrade kubelet
+### Upgrade kubelet and kubeproxy
 
 1.  From the Windows node, upgrade and restart the kubelet:
 
     ```powershell
     stop-service kubelet
-    curl.exe -Lo C:\k\kubelet.exe https://dl.k8s.io/{{< param "fullversion" >}}/bin/windows/amd64/kubelet.exe
+    curl.exe -Lo <path-to-kubelet>\kubelet.exe https://dl.k8s.io/{{< param "fullversion" >}}/bin/windows/amd64/kubelet.exe
     restart-service kubelet
     ```
 
+2. From the Windows node, upgrade and restart the kube-proxy.
+
+    ```powershell
+    stop-service kube-proxy
+    curl.exe -Lo <path-to-kubeproxy>\kube-proxy.exe https://dl.k8s.io/{{< param "fullversion" >}}/bin/windows/amd64/kube-proxy.exe
+    restart-service kube-proxy
+    ```
+
+{{< note >}}
+If you are running kube-proxy in a HostProcess Container instead of as a service, you can upgrade kube-proxy by applying a new version of kube-proxy manifests you are using.
```

Comment:
```suggestion
If you are running kube-proxy in a HostProcess Container instead of as a service, you can upgrade kube-proxy by applying a newer version of version of your kube-proxy deployment manifests.
```

nit

Reply:
fixed